core/src/main/java/org/elasticsearch/rest/action/admin/indices/analyze/RestAnalyzeAction.java

```diff
@@ -129,7 +132,20 @@ public static void buildFromContent(BytesReference content, AnalyzeRequest analy
                             }
                             charFilters.add(parser.text());
                         }
-                        analyzeRequest.tokenFilters(charFilters.toArray(Strings.EMPTY_ARRAY));
+                        analyzeRequest.charFilters(charFilters.toArray(Strings.EMPTY_ARRAY));
```

Comment:
I know that it comes from copy and paste of previous code but I think it's more efficient to do:

```java
analyzeRequest.charFilters(charFilters.toArray(new String[charFilters.size()]));
```